src/town/lilac/humble/app/gui_2.clj

```diff
@@ -41,7 +41,22 @@
   ;; => 41N
   (f->c 41)
   ;; => 5N
-  )
+  ,)
+
+(defn safe-parse-float
+  " behavior when text field is empty is undefined -- let's return 0.0 "
+  [s]
+  (if (or (nil? s)
+          (= s ""))
+    0.0
+    (Float/parseFloat s)))
+
+(comment
+  (safe-parse-float nil)   ;; => 0.0
+  (safe-parse-float "")    ;; => 0.0
+  (safe-parse-float "4.0") ;; => 4.0
+  0)
+
```

Comment:
this change seems to be different than #1 . can you merge you branch with main? I'll accept it then